File: Data Structures & Algorithms/is-anagram/submission-3.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] count = new int[26];

        if(s.length() != t.length()) return false;

        for(int i=0; i<s.length(); i++){
            char cs = s.charAt(i);
            char ct = t.charAt(i);

            count[cs - 'a']++;
            count[ct - 'a']--;
        }

        for(int i=0; i<count.length; i++){
            if(count[i] != 0) return false;
        }
        return true;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-3.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        seen = [0] * 1001
        nums.sort()
        if len(nums) <= 0:
            return 0

        # for num in nums:
        #     seen[abs(num)] += 1

        max_count = 1
        count = 1
        for R in range(len(nums) - 1):
            
            if nums[R] - nums[R + 1] == -1:
                count += 1
            elif nums[R] == nums[R + 1]:
                continue
            else:
                count = 1
            max_count = max(max_count, count)

        # for index, num in enumerate(seen):
        #     if num >= 1:
        #         count += 1
        #     else:
        #         count = 0
        #     max_count = max(max_count, count)
        #     print(f"{index}: {num}")
        return max_count
